lang/cs/Org.Apache.REEF.IO.TestClient/HadoopFileInputPartitionTest.cs

```diff
@@ -161,5 +163,26 @@ public IEnumerable<byte> Deserialize(string fileFolder)
                 }
             }
         }
+
+        /// <summary>
+        /// Read bytes from all the files in the set and return one by one
+        /// </summary>
+        /// <param name="filePaths"></param>
+        /// <param name="local"></param>
+        /// <returns></returns>
+        public IEnumerable<byte> Deserialize(ISet<string> filePaths, bool local)
```

Comment:
We want to return IEnumarable<T>. T can by Row, or int or string.... In this test, I just use byte as an example.